File: CpawCTF/Level1/Q6.[Crypto] Classical Cipher/solver.jl
c = "fsdz{Fdhvdu_flskhu_lv_fodvvlfdo_flskhu}"

for char in c
  # {}とか_はずらさないでそのまま出力する
  if all(isletter,char)
    print(Char(Int(char) - 3))
  else
    print(char) 
  end
end

File: CpawCTF/Level1/Q6.[Crypto] Classical Cipher/solver.py
c = 'fsdz{Fdhvdu_flskhu_lv_fodvvlfdo_flskhu}'

for char in c:
  # {}とか_はずらさないでそのまま出力する
  if char.isalpha():
    print(chr(ord(char) - 3),end='')
  else:
    print(char, end='')